### PR TITLE
Removed unused code, only deploy on master

### PR DESCRIPTION
**Pull request by UniversalSuperBox**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,22 @@
 sudo: required
 
 install:
-    - sudo bash -c "echo deb http://de.archive.ubuntu.com/ubuntu bionic main >> /etc/apt/sources.list"
+    - sudo bash -c "echo deb http://de.archive.ubuntu.com/ubuntu xenial main >> /etc/apt/sources.list"
+    - sudo bash -c "echo deb http://de.archive.ubuntu.com/ubuntu xenial universe >> /etc/apt/sources.list"
```
Comment (jbruechert) [inline .travis.yml]: Could this be done in one line?

File: build-initrd.sh [hunk 2]
```diff
@@ -5,6 +5,7 @@ set -e
 export FLASH_KERNEL_SKIP=1
 export DEBIAN_FRONTEND=noninteractive
 DEFAULTMIRROR="https://deb.debian.org/debian"
+APT_COMMAND="apt-get -y"
```
Comment (jbruechert) [inline build-initrd.sh]: As this is debian stretch, I think we could just use apt instead of apt-get

**Comment by UniversalSuperBox** [comment]
All done

**Comment by jbruechert** [comment]
Is it ok if I squash & merge this now?

**Comment by UniversalSuperBox** [comment]
Yep! All good.

On February 9, 2018 9:37:22 AM CST, JBB <notifications@github.com> wrote:
>Is it ok if I squash & merge this now?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/Halium/initramfs-tools-halium/pull/5#issuecomment-364468172

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Pull request merged**